backup_and_restore/application_backup_and_restore/oadp-advanced-topics.adoc

```diff
@@ -12,23 +12,31 @@ This document provides information about advanced features and functionalities o
 == Working with different Kubernetes API versions on the same cluster
 
```

Comment:
This is out of the scope of this PR, but this section needs a short introduction. Otherwise, you end up with one heading being displayed just after another heading.